### PR TITLE
On-boarding(plan-selected and verification)

### PR DESCRIPTION
**Pull request by priyanka9634**
On-boarding(plan-selected and verification)

### CODE REVIEW

File: lib.ftd [hunk 1]
```diff
@@ -6021,6 +6267,329 @@ right.px: 18
...
+-- ftd.text-input:
+padding.px: 6
+width: fill-container
+border-width.px: 1
+border-color: $inherited.colors.custom.two
+border-radius.px: 10
+min-height.fixed.px: 43
+margin-top.px: 8
+
+-- ftd.text: Card Holder Name
```
Comment (gsalunke) [inline lib.ftd:6311]:
```suggestion
-- ftd.text: <it should come from string property>
```

File: lib.ftd [hunk 2]
```diff
@@ -5904,6 +5906,133 @@ color: $inherited.colors.background.step-1
...
+-- ftd.text: $plan-selected.plan-select
+role: $fonts.font-16-21-700
+
+-- end: ftd.row
+
+-- ftd.row:
+spacing.fixed.px: 8
+role: $fonts.font-16-98-700
+
+-- ftd.text: \$
```
Comment (gsalunke) [inline lib.ftd:5971]:
```suggestion
-- ftd.text: <it should come from string property>
```
this seems to be currency and currency can be changed as per country code.

File: lib.ftd [hunk 3]
```diff
@@ -6021,6 +6267,329 @@ right.px: 18
...
+width.fixed.px: 16
+margin-right.px: 10
+
+-- ftd.text: Save this card for future use.
+role: $fonts.font-16-36-400
+color: $inherited.colors.text
+
+-- end: ftd.row
+
+-- ftd.text: Proceed
```
Comment (gsalunke) [inline lib.ftd:6404]:
```suggestion
-- ftd.text: <it should come from string property>
```
they can change button title, so it should come from property value

File: lib.ftd [hunk 4]
```diff
@@ -6021,6 +6267,329 @@ right.px: 18
...
+-- end: ftd.container
+
+-- end: ftd.column
+
+-- ftd.column:
+height: fill-container
+align-content: bottom-center
+width: fill-container 
+
+-- ftd.text: Submit
```
Comment (gsalunke) [inline lib.ftd:6495]:
```suggestion
-- ftd.text: <it should come from string property>
```

File: lib.ftd [hunk 5]
```diff
@@ -6021,6 +6267,329 @@ right.px: 18
...
+
+
+
+
+
+
+
+
+-- component input:
+
```
Comment (gsalunke) [inline lib.ftd:6521]:
```suggestion
string placeholder: Enter value
```

File: lib.ftd [hunk 6]
```diff
@@ -6021,6 +6267,329 @@ right.px: 18
...
+margin-top.px: 8
+align-content: center
+
+-- ftd.text-input:
+padding.px: 6
+width.fixed.px: 60
+min-height.fixed.px: 52
+border-color: $inherited.colors.custom.two
+border-radius.px: 10
+border-width.px: 1
```
Comment (gsalunke) [inline lib.ftd:6532]:
```suggestion
border-width.px: 1
placeholder: $input.placeholder
```

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: lib.ftd (hunk 2 above)
Comment (priyanka9634) [reply lib.ftd:5971]: $ is  for currency but we access each component property with $component.property-name. if we define a string property whose value is $ , it try to find variable which throws error

**Review by gsalunke — APPROVED** [review]

**Pull request merged**